src/OrchardCore.Modules/OrchardCore.Media/README.md

```diff
@@ -179,19 +179,29 @@ The following configuration values are used by default and can be customized:
 
 ```json
     "OrchardCore.Media": {
+
       // The accepted sizes for custom width and height
       "SupportedSizes": [ 16, 32, 50, 100, 160, 240, 480, 600, 1024, 2048 ],
 
-      // The number of days to store images in the browser cache
+      // The number of days to store images in the browser cache.
       "MaxBrowserCacheDays": 30,
 
-      // The number of days to store images in the image cache
+      // The number of days a cached resized media item will be valid for, before being rebuilt on request.
       "MaxCacheDays": 365,
 
       // The maximum size of an uploaded file in bytes. 
       // NB: You might still need to configure the limit in IIS (https://docs.microsoft.com/en-us/iis/configuration/system.webserver/security/requestfiltering/requestlimits/)
       "MaxFileSize": 30000000,
 
+      // A Cdn base url that will be prefixed to the request path when serving images.
+      "CdnBaseUrl": "https://your-cdn.com",
+
+      // The path used when serving media assets.
+      "AssetsRequestPath": "/media",
+
+      // The path in the tenant's App_Data folder containing the assets.
```

Comment:
We should mention that it can be an absolute path if we don't want it to be in App_Data

Reply:
Based on the current (existing) code it will always be combined under whatever App_Data Path is configured
``` 
private string GetMediaPath(ShellOptions shellOptions, ShellSettings shellSettings, string assetsPath)
{
  return PathExtensions.Combine(shellOptions.ShellsApplicationDataPath, shellOptions.ShellsContainerName, shellSettings.Name, assetsPath);
}
```

We could change that? Check for an absolute path, and not append perhaps?

Reply:
I don't think so, `PathExtensions` uses `Path.Combine` and if there is a `/` in one segment it will override the previous ones. We do it in `ShellOptionsSetup` this way for `App_Data`, so you can define it to another location than under the content root. 

https://dotnetfiddle.net/Yhurip

Reply:
Ah cool, had never looked at `PathExtensions`. Done. 

But noting that `PathExtensions` doesn't use `Path.Combine`, it just looks for a `/` or `\`, so works for absolute, but not for fully qualified. `ShellOptionsSetup` uses `Path.Combine`